test/quantization/core/test_workflow_module.py

```diff
@@ -1102,6 +1108,47 @@ def test_fused_mod_reduce_range(self):
         self.assertEqual(obs.quant_min, 0)
         self.assertEqual(obs.quant_max, 127)
 
+    def test_embedding_bag_qat_config(self):
+        class Model(nn.Module):
+            def __init__(self):
+                super(Model, self).__init__()
+                self.emb1 = torch.nn.EmbeddingBag(num_embeddings=10, embedding_dim=12,
+                                                  include_last_offset=True, scale_grad_by_freq=False, mode='sum')
+                self.emb2 = torch.nn.EmbeddingBag(num_embeddings=10, embedding_dim=12,
+                                                  include_last_offset=True, scale_grad_by_freq=False, mode='sum')
+
+            def forward(self, indices):
+                return torch.cat((self.emb1(indices), self.emb2(indices)))
+
+        model = Model()
+        indices = torch.randint(0, 10, (5, 12))
+        qat_mappings = get_default_qat_module_mappings()
+        qat_mappings[nn.EmbeddingBag] = torch.nn.qat.EmbeddingBag
+
+        float_qparams_observer = PerChannelMinMaxObserver.with_args(dtype=torch.quint8,
+                                                                    qscheme=torch.per_channel_affine_float_qparams,
+                                                                    ch_axis=0)
+        float_qparams_qconfig = QConfigDynamic(activation=default_dynamic_quant_observer,
+                                               weight=float_qparams_observer)
```

Comment:
For QAT you'd need a FakeQuantize module attached to the module being quantized. In this case, looks like you're only using the observer so i dont think the fake_quantize call will get invoked. Example of a QAT config https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/qconfig.py#L125

Reply:
More specifically for Embeddings we need to use a FakeQuantize module that is applied to weights that uses the `PerChannelMinMaxObserver` and the `torch.per_channel_affine_float_qparams` qscheme to compute the scale/zero_point. For activations we don't need a fake quantize.

Reply:
We actually need to use a stateless `PerChannelMinMaxObserver` (since the current observer uses a running min/max of incoming tensors). we can probably configure the observer base class to add an option to make it dynamic (stateless). cc @jerryzh168 since this also aligns with the task to enable dynamic quantization for QAT Linear.

Reply:
cc @HDCharles This should be similar to your ongoing stateless observer work for quantization